### ISSUE TITLE
Git Bash not showing

### ISSUE THREAD

**Issue by MrChriZ** [madskristensen/OpenCommandLine#42]
I have Bash installed but it's not showing up.

It's definitely installed in "C:\Program Files\git\git-bash.exe"

**Comment by mhagesfeld** [comment]
I am suffering the same issue - it just stopped working one day.  Were you able to figure out a conflict?

**Comment by MrChriZ** [comment]
I don't believe I got to the bottom of it.  I'm not using VS much at present so can't offer much else I'm afraid.

**Comment by mhagesfeld** [comment]
Found my issue - Git v2.13 installed bash in C:\Users\[user]\AppData\Local\Programs\Git\git-bash.exe

Once I switched that it worked, and I know that did not address your issue.
